server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -242,6 +253,8 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeLong(previousSectorsRead);
             out.writeLong(currentSectorsWritten);
             out.writeLong(previousSectorsWritten);
+            out.writeLong(currentIOTime);
+            out.writeLong(previousIOTime);
```

Comment:
We'll need to read/write the new fields in a backwards compatible way (similar to how we do it here for eg. https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/datafeed/AggProvider.java#L137 - there are probably other areas in the code). Let me know if you need any help with these version checks to ensure these fields are added in a BWC way.

Reply:
Thanks for the detailed instructions, i added  the compatibility checks ,use Version.V_7_13_0